perfmetrics/scripts/continuous_test/gcp_ubuntu/e2e_tests/build.sh

```diff
@@ -25,6 +25,9 @@ echo "Building and installing gcsfuse..."
 commitId=$(git log origin/$BRANCH_NAME --before='yesterday 23:59:59' --max-count=1 --pretty=%H)
 ./perfmetrics/scripts/build_and_install_gcsfuse.sh $commitId
 
+# To execute tests for a specific branch, ensure you've checked out from that branch first.
+git checkout $BRANCH_NAME
```

Comment:
don't we need origin/$BRANCH_NAME ?

Reply:
Git checkout does not require to define origin, I have tested. Although added it for safe side.